dashboards-observability/.cypress/integration/event_analytics.spec.js

```diff
@@ -615,4 +663,276 @@ describe('Renders data view', () => {
+    cy.get('#configPanel__panelOptions .euiFieldText').click().type('Line Chart');
+    cy.get('.euiFlexItem .euiFormRow [placeholder="Description"]').click().type('This is the description for Line chart with chart style of Bars');
+    cy.get('[data-text="Bars"]').should('have.text', 'Bars').click();
+    cy.get('[data-text="Center"] [data-test-subj="center"]').should('have.attr', 'checked');
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click({ force: true });
+  });
+
+  it('Render line chart and Verify Chart Style of Bars Mode with Before bar alignment ', () => {
+    cy.get('#configPanel__panelOptions .euiFieldText').click().type('Line Chart');
+    cy.get('.euiFlexItem .euiFormRow [placeholder="Description"]').click().type('This is the description for Line chart with chart style of Bars');
+    cy.get('[data-text="Bars"]').should('have.text', 'Bars').click();
+    cy.get('[data-text="Before"] [data-test-subj="before"]').should('not.have.attr', 'checked');
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click({ force: true });
+  });
+
+  it('Render line chart and Verify Chart Style of Bars Mode with After bar alignment ', () => {
+    cy.get('#configPanel__panelOptions .euiFieldText').click().type('Line Chart');
+    cy.get('.euiFlexItem .euiFormRow [placeholder="Description"]').click().type('This is the description for Line chart with chart style of Bars');
+    cy.get('[data-text="Bars"]').should('have.text', 'Bars').click();
+    cy.get('[data-text="After"] [data-test-subj="after"]').should('not.have.attr', 'checked');
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click({ force: true });
+  });
+
+  it('Render line chart and Verify Chart Style of Points Mode', () => {
+    cy.get('#configPanel__panelOptions .euiFieldText').click().type('Line Chart');
+    cy.get('.euiFlexItem .euiFormRow [placeholder="Description"]').click().type('This is the description for Line chart with chart style of Points');
+    cy.get('[data-text="Points"]').should('have.text', 'Points').click();
+    cy.get('[data-text="Points"] [data-test-subj="markers"]').should('not.have.attr', 'checked');
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click({ force: true });
+  });
+
+  it('Render line chart and Verify Chart Style of Points Mode with larger Point size', () => {
+    cy.get('#configPanel__panelOptions .euiFieldText').click().type('Line Chart');
+    cy.get('.euiFlexItem .euiFormRow [placeholder="Description"]').click().type('This is the description for Line chart with chart style of Points');
+    cy.get('[data-text="Points"]').should('have.text', 'Points').click();
+    cy.get('[data-text="Points"] [data-test-subj="markers"]').should('not.have.attr', 'checked');
+    cy.get('input[type="range"]')
+       .then($el => $el[0].stepUp(30))
+       .trigger('change')
+    cy.get('.euiRangeSlider').should('have.value', 35)
```

Comment:
check if it should be `30` as it is being set as `30`

Reply:
The Default value gets selected for Point size as 5 and we are moving slider to 30 step ahead then validating it for value 35